### PR TITLE
feat: In wall dimmer with new manufacturer id

### PR DESCRIPTION
**Pull request by fiLLLip**
The in wall dimmer has changed manufacturer id, so I copy pasted the
code and updated the new manufacturer id.

### CODE REVIEW

**Comment by fiLLLip** [comment]
> Briefly looked over. Looks OK to me.

Wow, that was fast!

**Comment by fiLLLip** [comment]
Solves #15

**Comment by aasmundbo** [comment]
In these Covid-19 times ... :)

Edit: DYAC

**Comment by fiLLLip** [comment]
Did some refactoring in the light of comment https://github.com/fiLLLip/namron-homey/issues/16#issuecomment-600235517

File: app.json [hunk 1]
```diff
@@ -197,7 +197,10 @@
         "small": "/drivers/ZW_Dimmer_3wire/assets/images/small.png"
       },
       "zwave": {
-        "manufacturerId": 816,
+        "manufacturerId": [
```
Comment (aasmundbo) [inline app.json:200]: Much better as a list/array if it works!

**Comment by fiLLLip** [comment]
Solves #10

**Pull request merged**